app/app.rb

```diff
@@ -28,6 +30,21 @@ class MakersBnb < Sinatra::Base
     erb(:"/space/view")
   end
 
+  get '/user/new' do
+    erb(:"user/new")
+  end
+
+  post '/user/new' do
+    @user = User.create(name: params[:name])
+    session[:user_id] = @user.id
+    redirect '/space/new'
+  end
```

Comment:
do we need instance var here? (use local var instead of instance var where possible)